test/unit/omp-target/test-reduce-tuplemaxloc.cpp

```diff
@@ -124,15 +124,11 @@ TYPED_TEST_P(ReductionTupleLocTestTargetOMP, ReduceMaxLocIndex)
 
   auto actualdata = this->data;
   auto indirect = this->array;
-  // TODO: remove this when compilers (clang-coral and IBM XLC) are no longer
-  // broken for lambda capture
-#pragma omp target data use_device_ptr(actualdata)
-#pragma omp target data use_device_ptr(indirect)
   RAJA::forall<ExecPolicy>(rowrange, [=] (int r) {
-    RAJA::forall<ExecPolicy>(colrange, [=] (int c) {
-      // TODO: Clang compiles but seg faults.
-      maxloc_reducer.maxloc(indirect[r][c], Index2D(c, r));
-    });
+    for(int c : colrange) {
```

Comment:
Good catch here and below. We should never be nesting foralls.

Reply:
Right, I'm not sure how that ever worked, but it was dying on the new plugin invocation calls.

Reply:
Sorry about the nested foralls . . .